README.md

```diff
@@ -151,3 +152,33 @@ Use npm modules with node that mimic the same tasks but are system agnostic. Som
 * set environment variable (`ENV_VAR = ...`) -> [`cross-env`](https://www.npmjs.com/package/cross-env)
 
 [↑ back to Table of Contents](#table-of-contents)
+
+## Document your script API
+
+### Why?
+
+* Documentation provides developers with a high level overview to the script, without the need to go through all its code. This makes a module more accessible and easier to use.
+
+* Documentation formalises the API.
+
+* README.md is the de facto standard filename for documentation to be read first. Code repository hosting services (Github, Bitbucket, Gitlab etc) display the contents of the the README's, directly when browsing through source directories. This applies to our module directories as well.
```

Comment:
The whole "README.md is the de facto standard ..." is a bit long. Maybe move it to the *How?* saying "Document your script API in the project's README.md or CONTRIBUTING.md as those are the first places contributors will look."